src/player_activity.cpp

```diff
@@ -70,6 +73,22 @@ cata::optional<std::string> player_activity::get_progress_message() const
         if( const item *craft = targets.front().get_item() ) {
             extra_info = craft->tname();
         }
+    } else if( type == activity_id( "ACT_READ" ) ) {
+        if( const item *book = targets.front().get_item() ) {
+            if( const auto &reading = book->type->book ) {
+                const skill_id &skill = reading->skill;
+                if( skill && u.get_skill_level( skill ) < reading->level &&
+                    u.get_skill_level_object( skill ).can_train() ) {
+                    const SkillLevel &skill_level = u.get_skill_level_object( skill );
+                    //~ skill_name current_skill_level -> next_skill_level (% to next level)
+                    extra_info = string_format( _( "%s %d -> %d (%d%%)" ),
```

Comment:
```suggestion
                    extra_info = string_format( pgettext( "reading progress", "%s %d -> %d (%d%%)" ),
```
To avoid conflict if another string with a different context happens to be identical to this one.